Source/SIMPLib/Geometry/ImageGeom.cpp

```diff
@@ -235,7 +234,7 @@ class FindImageDerivativesImpl
   }
 #endif
 
-  void computeIndices(int32_t differenceType, int32_t directionType, size_t& index1, size_t& index2, size_t dims[3], size_t x, size_t y, size_t z, double xp[3], double xm[3]) const
+  void computeIndices(int32_t differenceType, int32_t directionType, size_t& index1, size_t& index2, const size_t dims[3], size_t x, size_t y, size_t z, double xp[3], double xm[3]) const
```

Comment:
Because dims[3] has a const, I'm going to ask for confirmation that xm[3] and xp[3] are expected to have their values updated and "returned" by this method.  Otherwise, the inconsistency is misleading.

Reply:
I can confirm the xm and xp are modified in this function; they are ultimately passed into a call to getCoords() which will set their values to the proper (x,y,z) coordinates depending on the derivative stencil.